mypy/stubtest.py

```diff
@@ -1486,6 +1486,9 @@ def verify_typealias(
         "__instancecheck__",
         "__subclasshook__",
         "__subclasscheck__",
+        # PEP 695 implementation details
+        "__annotate__",
```

Comment:
`__annotate__` is from PEP 649, not 695. `__classdictcell__` does come from PEP 695 (so it's already in 3.12+), but is used only during class body execution and I don't think stubtest should see it. Did you encounter problems with it?